backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/UserMutationResolver.java

```diff
@@ -27,102 +28,114 @@ public UserMutationResolver(ApiUserService us) {
   }
 
   @AuthorizationConfiguration.RequireGlobalAdminUser
+  @MutationMapping
   public User addUser(
-      PersonName name,
-      String firstName,
-      String middleName,
-      String lastName,
-      String suffix,
-      String email,
-      String organizationExternalID,
-      Role role) {
+      @Argument PersonName name,
+      @Argument String firstName,
+      @Argument String middleName,
+      @Argument String lastName,
+      @Argument String suffix,
+      @Argument String email,
+      @Argument String organizationExternalId,
+      @Argument Role role) {
     name = Translators.consolidateNameArguments(name, firstName, middleName, lastName, suffix);
-    UserInfo user = _us.createUser(email, name, organizationExternalID, role);
+    UserInfo user = _us.createUser(email, name, organizationExternalId, role);
     return new User(user);
   }
 
+  @MutationMapping
   public User addUserToCurrentOrg(
-      PersonName name,
-      String firstName,
-      String middleName,
-      String lastName,
-      String suffix,
-      String email,
-      Role role) {
+      @Argument PersonName name,
+      @Argument String firstName,
+      @Argument String middleName,
+      @Argument String lastName,
+      @Argument String suffix,
+      @Argument String email,
+      @Argument Role role) {
     name = Translators.consolidateNameArguments(name, firstName, middleName, lastName, suffix);
     UserInfo user = _us.createUserInCurrentOrg(email, name, role, true);
     return new User(user);
   }
 
+  @MutationMapping
   public ApiUser createApiUserNoOkta(
-      PersonName name,
-      String firstName,
-      String middleName,
-      String lastName,
-      String suffix,
-      String email) {
+      @Argument PersonName name,
+      @Argument String firstName,
+      @Argument String middleName,
+      @Argument String lastName,
+      @Argument String suffix,
+      @Argument String email) {
     name = Translators.consolidateNameArguments(name, firstName, middleName, lastName, suffix);
     return _us.createApiUserNoOkta(email, name);
   }
 
+  @MutationMapping
   public User updateUser(
-      UUID id,
-      PersonName name,
-      String firstName,
-      String middleName,
-      String lastName,
-      String suffix) {
+      @Argument UUID id,
+      @Argument PersonName name,
+      @Argument String firstName,
+      @Argument String middleName,
+      @Argument String lastName,
+      @Argument String suffix) {
     name = Translators.consolidateNameArguments(name, firstName, middleName, lastName, suffix);
     UserInfo user = _us.updateUser(id, name);
     return new User(user);
   }
 
-  // Making `facilities` an array instead of a Collection to avoid type erasure
```

Comment:
I assume new graphQL has fixed this UUID issue?

Reply:
Yes, they have. Matter fact primitive types don't work that great on the arguments